.sync.yml

```diff
@@ -10,5 +10,8 @@ spec/spec_helper.rb:
     - name: root_home
       value: /root
       source: puppetlabs-stdlib
+    - name: service_provider
```

Comment:
It was my intention to move to `add_mocked_facts!` (from [voxpupuli-test](https://github.com/voxpupuli/voxpupuli-test/blob/master/lib/voxpupuli/test/facts.rb)) but haven't gotten around to it. We should implement this in our modulesynced `spec_helper.rb`. It would make all these custom facts redundant.